cmd/course-user-get/main_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/edulinq/autograder/internal/cmd"
+)
+
+// Use the common main for all tests in this package.
+func TestMain(suite *testing.M) {
+	cmd.CMDServerTestingMain(suite)
+}
+
+func TestCourseUserGetBase(test *testing.T) {
+	testCases := []struct {
+		cmd.CommonCMDTestCase
+		targetEmail string
+		courseID    string
+	}{
+		{
+			CommonCMDTestCase: cmd.CommonCMDTestCase{
+				ExpectedStdout: USER_COURSE_FOUND,
+			},
+			targetEmail: "course-student@test.edulinq.org",
+			courseID:    "course101",
+		},
+		{
+			CommonCMDTestCase: cmd.CommonCMDTestCase{
+				ExpectedStdout: USER_NOT_FOUND,
+			},
+			targetEmail: "ZZZ",
+			courseID:    "course101",
+		},
```

Comment:
Add one more test case for a user that exists in the database, but is not enrolled in the course.

See the python interface test case for reference (https://github.com/edulinq/autograder-py/blob/main/tests/api/testdata/courses/users/courses_users_get_not_enrolled.json).